### ISSUE TITLE
EF 6.3.0-preview5 with .NET Core 3.0.0-preview5 30 fields limit?

### ISSUE THREAD

**Issue by epic2001** [aspnet/EntityFramework6#837]
I tried EF 6.3.0-preview5 with .NET Core 3.0.0-preview5 but I get an Exception when I try to get data from a table with more than 30 fields.
I get the Exception only when the POCO entity class have more then 30 fields defined.
If I comment any field property in the POCO entity class (do not matter which): under 30 fields it works. Over 30 fields the code do not works.

```
Inner Exception: IndexOutOfRangeException: Index was outside the bounds of the array.
Stack trace:
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition..ctor(DbProviderFactory storeProviderFactory, DbCommandTree commandTree, DbInterceptionContext interceptionContext, IDbDependencyResolver resolver, BridgeDataReaderFactory bridgeDataReaderFactory, ColumnMapFactory columnMapFactory)
   at System.Data.Entity.Core.EntityClient.Internal.EntityProviderServices.CreateDbCommandDefinition(DbProviderManifest providerManifest, DbCommandTree commandTree, DbInterceptionContext interceptionContext)
   at System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlanFactory.CreateCommandDefinition(ObjectContext context, DbQueryCommandTree tree)
   at System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlanFactory.Prepare(ObjectContext context, DbQueryCommandTree tree, Type elementType, MergeOption mergeOption, Boolean streaming, Span span, IEnumerable`1 compiledQueryParameters, AliasGenerator aliasGenerator)
   at System.Data.Entity.Core.Objects.ELinq.ELinqQueryState.GetExecutionPlan(Nullable`1 forMergeOption)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass41_0.<GetResults>b__1()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass41_0.<GetResults>b__0()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__31_0()
   at System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at ConsoleApp2.Program.Main(String[] args) in C:\\Users\\stefa\\source\\Workspaces\\Perizie\\Perizie\\ConsoleApp2\\Program.cs:line 14
```

### Steps to reproduce

**Class Library (.NET Framework)**

DbContext01.cs

```c#
    public class DbContext01 : DbContext
    {
        public DbContext01(string connectionString) : base(connectionString) { }
        public DbSet<Table01> Table01 { get; set; }
        protected override void OnModelCreating(DbModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);
        }
    }
```

Table01.cs

```c#
    public class Table01
    {
        [Key]
        public int Field01 { get; set; }
        public int Field02 { get; set; }
        public int Field03 { get; set; }
        public int Field04 { get; set; }
        public int Field05 { get; set; }
        public int Field06 { get; set; }
        public int Field07 { get; set; }
        public int Field08 { get; set; }
        public int Field09 { get; set; }
        public int Field10 { get; set; }
        public int Field11 { get; set; }
        public int Field12 { get; set; }
        public int Field13 { get; set; }
        public int Field14 { get; set; }
        public int Field15 { get; set; }
        public int Field16 { get; set; }
        public int Field17 { get; set; }
        public int Field18 { get; set; }
        public int Field19 { get; set; }
        public int Field20 { get; set; }
        public int Field21 { get; set; }
        public int Field22 { get; set; }
        public int Field23 { get; set; }
        public int Field24 { get; set; }
        public int Field25 { get; set; }
        public int Field26 { get; set; }
        public int Field27 { get; set; }
        public int Field28 { get; set; }
        public int Field29 { get; set; }
        public int Field30 { get; set; }
        public int Field31 { get; set; }
    }
```

**App Console (.NET Core)**

```c#
    class Program
    {
        static void Main(string[] args)
        {
            DbProviderFactories.RegisterFactory("System.Data.SqlClient", SqlClientFactory.Instance);
            var db = new DbContext01(@"Server=.\SQLEXPRESS;Database=Test01;Trusted_Connection=True;MultipleActiveResultSets=true");
            var result = db.Table01.FirstOrDefault();
        }
    }
```

### Further technical details
EF version: 6.3.0-preview5
Database Provider: (EntityFramework.SqlServer)
Operating system: (Windows 10) 
IDE: (Visual Studio 2019 Preview)

**Comment by epic2001** [comment]
I forgot to say that the problem don't appear when the console app is .net framework instead of .net core.

**Comment by robertsundstrom** [comment]
I have experienced the same issue.

**Comment by bricelam** [comment]
Looks like this had the same root cause as #861. It was an external issue that will be fixed in Preview 6: dotnet/corefx#37849
